source/bisect.ts

```diff
@@ -0,0 +1,54 @@
+import {OptionsSync} from 'webext-options-sync-per-domain';
+
+import {RGHOptions, perDomainOptions} from './options-storage';
+
+export default async function bisectFeatures(): Promise<void> {
+	const options = perDomainOptions.getOptionsForOrigin();
+	const featuresState = Object.fromEntries(Object.entries(await options.getAll()).filter(([key]) => key.startsWith('feature:')));
```

Comment:
What do you think about moving this whole logic to the content script?

Advantage:

- the user doesn't need to juggle 2 windows (GitHub.com and the options)
- the popup appears wherever the user is and gives exact feedback on whether it's bisecting the current view or not

Disadvantage:

- we have to build the UI instead of relying on alert/confirm

I can help with the style of this UI, you can start with something like this inside `globalReady`:

```jsx
if (await cache.get('bisect')) {
	  document.body.append(
		  <div className="some GH class for background and border" style={{position: fixed, top: 0, left: 0}}>
		    {x} steps remaining. Can you see the bug?
		    <button onClick={onYes}>Yes</button> <!-- or a single {onClick} function -->
		    <button onClick={onNo}>No</button>
		  </div>
	  );
}
```

And then in the options page you just need this:

```js
async function bisectFeaturesHandler() {
	await cache.set('bisect', [], {minutes: 5});
	await browser.tabs.create({url: 'https://github.com'})
}
```